src/utils/format-date.js

```diff
@@ -79,3 +79,14 @@ export function dateHasPassed(unixTimestamp) {
 export function getCurrentDateTimestamp() {
   return Date.now() / 1000
 }
+
+export function getDaysRemaining(endTimestamp, startTimestamp) {
+  if (!endTimestamp) return 0
+  let start = startTimestamp
+  if (!startTimestamp) {
+    start = formatDate(new Date()).timestamp
+  }
+  if (start > endTimestamp) return 0
+  const remainingTicks = endTimestamp - start
+  return Math.floor(remainingTicks / 86400)
```

Comment:
For legibility, can you move this to a constant? Like NUMBER_OF_SECONDS_IN_A_DAY or AVERAGE_WAIT_FOR_BBQ_IN_AUSTIN. ;)

Reply:
Will add `NUMBER_OF_SECONDS_IN_A_DAY` as constant.